fsharp-backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -343,7 +341,7 @@ and Pattern =
 
 
 
-
+// TODO: this seems to be redundant with separate Shortcuts.fs file
```

Comment:
@pbiggar can/should we delete one of these Shortcuts modules?

Reply:
Shortcuts.fs is for RuntimeTypes, this is for ProgramTypes. I agree a refactor is in order! We may be able to get rid of ProgramTypes shortcuts as we have the "parser" now.

Reply:
I tried deleting and going down this route, but wasn't confident how to replace the usage within `TestUtils/FSharpToExpr`, so created a separate Issue to consider the path forward here.